### ISSUE TITLE
make package error

### ISSUE THREAD

**Issue by Cregin** [ipadkid358/BJUtils#1]
> Making all for tweak BJUtils…
==> Compiling BJLocation.m (arm64)…
==> Compiling BJSBAlertItem.m (arm64)…
==> Compiling BJServer.m (arm64)…
BJServer.m:99:175: error: property 'name' not found on object of type
      'CLPlacemark *'
  ...target=\"_blank\">(%f, %f)</a>", targetPlacemark.name, targetPlacemark.l...
                                                      ^
1 error generated.
make[3]: *** [/Users/macos/Downloads/BJUtils-master/.theos/obj/debug/arm64/BJServer.m.62349286.o] Error 1
make[2]: *** [/Users/macos/Downloads/BJUtils-master/.theos/obj/debug/arm64/BJUtils.dylib] Error 2
make[1]: *** [internal-library-all_] Error 2
make: *** [BJUtils.all.tweak.variables] Error 2

**Comment by ipadkid358** [comment]
This is due to Theos built-in [headers](https://github.com/theos/headers)

Thank you for opening this Issue, I overlooked the change I made to my personal copy of Theos. 

The [Theos header](https://github.com/theos/headers/blob/master/CoreLocation/CLPlacemark.h) is read before the target SDK. I will be opening an Issue on theos/headers to resolve this conflict in some way, as my project is the not the only one affected by these poorly stubbed out classes.

This issue will not be closed until my Issue is closed on the Theos repo(s)
